st2api/st2api/controllers/v1/triggers.py

```diff
@@ -244,10 +245,10 @@ def post(self, trigger):
             LOG.exception('Validation failed for trigger data=%s.', trigger)
             abort(http_client.BAD_REQUEST, str(e))
             return
-        except NotUniqueError as e:
+        except StackStormDBObjectConflictError as e:
```

Comment:
Down the road we could propagate those exceptions and have a centralized error handler which returns correct status code and body for each exception.

This way we don't need to have duplicated code all over the place and it makes maintenance and updating error responses a lot easier.